src/cli.rs

```diff
@@ -300,8 +300,8 @@ pub enum SshConfig {
         #[arg(short = 'u', long = "user", default_value = "omnect")]
         username: String,
         /// optional: path where the ssh key pair, the certificates, and the
-        /// temporary ssh configuration is stored. Defaults to system local data
-        /// directories (e.g. ${XDG_RUNTIME_DIR}/omnect-cli on Linux).
+        /// temporary ssh configuration is stored. Defaults to system local
+        /// runtime directory (e.g. ${XDG_RUNTIME_DIR}/omnect-cli on Linux).
```

Comment:
Imho the comment is still wrong:
```
jan@JZ-Mint-I631:~$ echo ${XDG_RUNTIME_DIR}
/run/user/1000
```
vs. 
`~/.local/share/omnect-cli/config`

Reply:
ack, that one slipped through. It is fixed now.